v2/cf-nodejs-logging-support/src/lib/logger/cache.ts

```diff
@@ -0,0 +1,63 @@
+import { ConfigField, outputs } from "../config/interfaces";
+import { SourceUtils } from "./source-utils";
+
+export default class Cache {
+
+    private static instance: Cache;
+    private sourceUtils: SourceUtils;
+    private cacheMsgRecord: any;
+    private cacheReqRecord: any;
+    public shouldUpdateMsg: boolean;
+    public shouldUpdateReq: boolean;
```

Comment:
I think these attributes should be private. You can add methods to set them, e.g., `markReqCacheDirty()`.

Reply:
Nice idea, I added a method `markCacheDirty()` to set both attributes with one call.